src/components/response_error.tsx

```diff
@@ -8,13 +8,6 @@ import Tile from './tile';
 // utils
 import { I18n } from '../utils/i18n';
 
-// a mapping of error codes to translated error messages which may include
-// the error parameters as replacement keys
-export type ErrorMessageTemplates = { [key: string]: string };
```

Comment:
Why not change this to `{ [key: string]: () => string },` here? We can share this for both error messages and fields, e.g. 
`export type TranslationTemplates = { [key: string]: () => string }`

Reply:
It's not used that often and this reduces a 'dependency' on an import from react-web-components from the generated files, which source code lives in commons (and thus can become outdated quickly).

Reply:
I do prefer to have it though otherwise we don't know when stuff is broken